cypress/integration/org_events_page.spec.ts

```diff
@@ -1,22 +1,51 @@
 describe('/o/[orgId]/events', () => {
+    beforeEach(() => {
+        cy.request('delete', 'http://localhost:8001/_mocks');
+    });
 
     it('contains name of organization', () => {
         cy.visit('/o/1/events');
         cy.contains('My Organization');
     });
 
-    xit('contains events which are linked to event pages', () => {
-        // TODO: Figure out why this fails on GitHub
-        cy.visit('/o/1/events');
-        cy.get('[data-test="event"]')
-            .eq(1)
-            .findByText('More info')
-            .click();
-        cy.url().should('match', /\/o\/1\/events\/22$/);
+    it.only('contains events which are linked to event pages', () => {
+        cy.request('put', 'http://localhost:8001/v1/orgs/1/campaigns/_mocks/get', {
+            response: {
+                data: {
+                    data: [{
+                        id: 2,
+                        title: 'The campaign',
+                    }],
+                },
+            },
+        });
+
+        cy.fixture('dummyEvents').then(json => {
+            cy.request('put', 'http://localhost:8001/v1/orgs/1/campaigns/2/actions/_mocks/get', {
+                response: {
+                    data: json,
+                },
+            });
+
+            cy.visit('/o/1/events');
+            cy.waitUntilReactRendered();
+            cy.get('[data-test="event"]')
+                .eq(0)
+                .findByText('More info')
+                .click();
+            cy.url().should('match', new RegExp(`/o/1/events/${json.data[0].id}$`));
+        });
     });
 
-    xit('contains a placeholder if there are no events', () => {
-        // TODO: Figure out why this fails on GitHub
+    it('contains a placeholder if there are no events', () => {
+        cy.request('put', 'http://localhost:8001/v1/orgs/1/campaigns/1/actions/_mocks/get', {
```

Comment:
This test doesn't actually try to find any placeholder?

Reply:
Yes, I believe it does, on line 52.

Although I found another issue revisiting the code now, which is that the old intercept is still left behind and should be removed.

Reply:
ah ok! unfmailiar with cypress and must have missed it. Looks good!